plugin/src/main/java/org/opensearch/ml/action/deploy/TransportDeployModelAction.java

```diff
@@ -285,10 +285,10 @@ private void deployModel(
             String taskId = response.getId();
             mlTask.setTaskId(taskId);
             if (algorithm == FunctionName.REMOTE) {
-                if (mlFeatureEnabledSetting.isMultiTenancyEnabled()) {
-                    listener.onResponse(new MLDeployModelResponse(taskId, MLTaskType.DEPLOY_MODEL, MLTaskState.CREATED.name()));
-                    return;
-                }
+                // if (mlFeatureEnabledSetting.isMultiTenancyEnabled()) {
+                // listener.onResponse(new MLDeployModelResponse(taskId, MLTaskType.DEPLOY_MODEL, MLTaskState.CREATED.name()));
+                // return;
+                // }
```

Comment:
Why are these commented? Remove them if they are no longer required.

Reply:
Yeah, I meant to remove. I'll remove this in next rev